includes/installer.php

```diff
@@ -128,6 +128,7 @@ public function get_database_schema() {
             currency_code varchar(3) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL DEFAULT 'USD',
             amount decimal(10,2) NOT NULL DEFAULT 0.00,
             status varchar(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
+            is_test boolean NULL DEFAULT FALSE,
```

Comment:
I'm thinking we might need to not have a default here. If we do, we won't have a way of knowing if the payment was not in test mode, or if it was created before the DB change.

Reply:
That makes sense yes. I will update it.